packages/babel-types/src/builders/index.ts

```diff
@@ -0,0 +1,5 @@
+import { numericLiteral, unaryExpression } from "./generated";
+
+export function buildUndefinedNode() {
+  return unaryExpression("void", numericLiteral(0), true);
+}
```

Comment:
Can we rename this file to `builders/productions.ts`? In this case the `index.ts` is for aggregating all the exports from sub modules: `flow`, `generated`, `react`, `productions`.

We can add more productions as well, such as `!0` as `true` literal and `!1` as `false`.

Reply:
`packages\babel-types\src\index.ts`
Unfortunately currently we aggregate the exports of all submodules in this file.🤦‍♂️
It would be a bit inconsistent if we just changed this.